packages/node-resolve/test/root-dir.js

```diff
@@ -0,0 +1,25 @@
+const { join } = require('path');
+
+const test = require('ava');
+const { rollup } = require('rollup');
+
+const { testBundle } = require('../../../util/test');
+
+const nodeResolve = require('..');
+
+process.chdir(join(__dirname, 'fixtures', 'monorepo-dedupe', 'packages', 'package-a'));
+
+test.only('deduplicates modules from the given root directory', async (t) => {
```

Comment:
Using `only` will make it harder to add other tests to this file later. Does either `test` or `test.serial` work?

Reply:
Indeed, it should be just `test`. Updated it!